client/src/components/NavBar.js

```diff
@@ -131,7 +131,7 @@ const Navigation = (props) => {
   const handleDropdownClick = (route) => () => {
     if (route === '/logout') {
       props.setAuthenticated(false);
-      props.setUserDetails(null, '');
+      props.setUserDetails(null, '', '', '', '');
```

Comment:
Don't worry too much about this in this ticket, but there probably is a better way of doing this, (and yes, I know i did the original :P)